server/controllers/post.controller.js

```diff
@@ -0,0 +1,81 @@
+const { where, Op } = require('sequelize')
+const db = require('../models')
+const Post = db.posts
+
+exports.deleteOnePost = async (req, res) => {
+    await Post.destroy({ where: [{ user_id: res.locals.userID }, { id: req.body.id }] })
+        .then(data => {
+            res.send('deleted');
+        })
+        .catch(err => {
+            res.status(500).send({
+            message:
+                err.message || "Some error occurred while creating the Tutorial."

```

Comment:
Please take a look on Google how to handle errors according to different status codes.
You only use 500 error code, but this is not true. Learn more about the status codes in more detail